src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndProjData.cxx

```diff
@@ -1063,6 +1133,28 @@ void RPC_process_related_viewgrams_accumulate_loglikelihood(
                              rim_truncation_sino, log_likelihood_ptr);
 };      
 
+void RPC_process_related_viewgrams_sensitivity_computation(
+                                                            const shared_ptr<ForwardProjectorByBin>& forward_projector_sptr,
+                                                            const shared_ptr<BackProjectorByBin>& back_projector_sptr,
+                                                            DiscretisedDensity<3,float>* output_image_ptr,
+                                                            const DiscretisedDensity<3,float>* input_image_ptr,
+                                                            RelatedViewgrams<float>* measured_viewgrams_ptr,
+                                                            int& count, int& count2, double* log_likelihood_ptr,
+                                                            const RelatedViewgrams<float>* additive_binwise_correction_ptr,
+                                                            const RelatedViewgrams<float>* mult_viewgrams_ptr)
+{
+
+  assert(output_image_ptr != NULL);
+  assert(measured_viewgrams_ptr != NULL);
+
+  RelatedViewgrams<float> viewgrams = measured_viewgrams_ptr->get_empty_copy();
```

Comment:
this is ok but somewhat wasteful in the (most useful) case where mul_viewgrams_ptr is not 0. I suggest something like

```
if( mult_viewgrams_ptr )
    back_projector_sptr->back_project(*output_image_ptr, *mult_viewgrams_ptr);
else
{
  RelatedViewgrams<float> viewgrams = measured_viewgrams_ptr->get_empty_copy();
  viewgrams.fill(1.F);
  back_projector_sptr->back_project(*output_image_ptr, viewgrams);
}
```